aopy/analysis/base.py

```diff
@@ -698,6 +702,78 @@ def get_sgram_multitaper(data, fs, win_t, step_t, nw=None, bw=None, adaptive=Fal
 
     return fxx, txx, Sxx
 
+def tfspec(X, n, p, k, fs, dn=None, fk=None, pad=2, ref=True):
```

Comment:
can you rename this to calc_mt_tfr()? to be similar to the existing function https://analyze.readthedocs.io/en/latest/analysis.html?highlight=tfr#aopy.analysis.base.calc_cwt_tfr

can you also change some of these to be more meaningful? e.g. "X" could be "ts_data", dn could be "window_size" or something like that, etc.